tests/test_data/output_examples/multiple_macros.cpp.preprocessed.cc

```diff
@@ -0,0 +1,6 @@
+// Macro FINALMACRO with args None expanding to '"this is the last macro"'
```

Comment:
should we keep a similar pattern with only using a single file extension?

Reply:
With the example filenames? We can set whatever pattern we want, this was just me being lazy with the naming scheme (just `mv`ing the output files)

Reply:
sounds good, i figured we would have intermediate files, i wasn't sure if you wanted to maintain adding on extensions instead just naming it ".o" or whatnot.